### DISCUSSION TITLE
Within the realm of psychology, what phenomenon describes . . .

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#72] [Galaxy Qs] [answered]
Within the realm of psychology, what phenomenon describes the tendency to attribute other people's behavior to internal factors while explaining our own behavior based on external factors, often leading to errors in judgment?

**Comment by mitchiemt11** [comment, marked as answer]
Fundamental Attribution error.

**Reply by Sriansh-raj** [reply]
Correct 🎉
